jme3-core/src/plugins/java/com/jme3/texture/plugins/DXTFlipper.java

```diff
@@ -202,7 +204,11 @@ private static void flipDXT1orDXTA3Block(byte[] block, int h){
         }
     }
 
-    public static ByteBuffer flipDXT(ByteBuffer img, int w, int h, Format format){
+    public static ByteBuffer flipDXT(ByteBuffer img, int w, int h, Format format) {
+        if (format == Format.ETC1 || format == Format.ETC2 || format == Format.ETC2_ALPHA1) {
+            logger.warning("This is not a DXT format, but ETC. Use flipETC instead.");
+            return ETCFlipper.flipETC(img, w, h, format);
```

Comment:
Why call ETCFlipper.flipETC(img, w, h, format); if it doesn't do anything? why not just return img?

Reply:
> ETC flipping is not supported: i couldn't figure out how to flip etc compressed textures without decompressing them first, even though it is probably possible (similar to dxt flipping). I don't this this is very important, since compressed textures are usually considered in their final form and they can be flipped during compression if needed, also we didn't even had dxt flipping for a long time. I plan to open an issue for this and hopefully someone that is knowledgeable about etc formats can contribute with the missing code one day.


Basically i don't know how to implement compressed etc flipping, so i wrote all the supporting code in the hope that someone can go into ETCFlipper some day and write the missing piece without the need to understand and modify the dds loader code that has some jme specific logic inside. 
We could open a bounty for it.